src/client/swagger/unity.yml

```diff
@@ -1277,10 +1254,35 @@ components:
         date:
           type: string
           description: date org was created
-        # TODO(ariel): this seems cyclical since accounts and orgs are subsets of one another
-        account:
-          $ref: '#/components/schemas/Account'
-      required: [id, idpeID, name, account, region, provider, date]
+        accountType:
+          type: string
+          description: type of the account
+          enum:
+            - free
+            - cancelled
+            - pay_as_you_go
+        accountBalance:
+          type: number
+          description: balance of the account
+        accountEmail:
+          type: string
+          description: billing contact email
+        accountId:
+          type: string
+          description: id of the account associated with the org
```

Comment:
besides the `accountType`, the rest of the account-specific data here is listed for the operator page's purposes. The data was flattened in order to remove the circular dependency that existed between accounts & orgs.

Reply:
What's the OEM API? Totally makes sense to have a RelatedAccount type that can be nested inside, it didn't even occur to me that it would be an option :)

Reply:
We have a public API used by OEMs like PTC and Schneider Electric. It allows them to add/remove orgs from their account (they have a single account, but create an org for each of their end customers) and do stuff with org limits.

Reply:
Awww, gotcha.